app/display.c

```diff
@@ -87,6 +87,37 @@ int max_cpu_temp = 0;
 
 display_mode_t display_mode = DISPLAY_MODE_NA;
 
+screen_palette_t mt_palette = {BLUE, WHITE, WHITE, BLACK, WHITE, BLUE, BLACK};
+
+//------------------------------------------------------------------------------
+// Private Functions
+//------------------------------------------------------------------------------
+
+void set_screen_palette(screen_palette_t *palette)
+{
+    if (dark_mode) {
+        *palette = (screen_palette_t){
+            .background        = BLACK,
+            .foreground        = WHITE,
+            .title_background  = WHITE,
+            .title_foreground  = BLACK,
+            .footer_background = WHITE,
+            .footer_foreground = BLACK,
+            .popup_background  = WHITE
+        };
+    } else {
+        *palette = (screen_palette_t){
+            .background        = BLUE,
+            .foreground        = WHITE,
+            .title_background  = WHITE,
+            .title_foreground  = BLACK,
+            .footer_background = WHITE,
+            .footer_foreground = BLUE,
+            .popup_background  = BLACK
+        };
+    }
+}
```

Comment:
If we convert this to a list of screen_palette_t[] and add a .name field with values like "classic" and "dark" then it would be super trivial to add another option in the future if we ever decide to do so.

By doing so, the initial palette (mt_palette) can simply point to entry [0] in this table so we can avoid re-peating these values in two independent places, one could even think abut adding a define for it into ./app/config.c instead of just using 0.